docs/api.rst

```diff
@@ -5290,7 +5290,7 @@ The following exceptions are thrown by the library.
 
 .. autoexception:: InvalidData
 
-.. autoexception:: InvalidArgument
+.. autoexception:: WebhookMissingToken
```

Comment:
Could call this `WebhookTokenMissing` instead, I feel like that would match the others better (`...Responded`, `...Supported`)

Reply:
https://github.com/DisnakeDev/disnake/pull/471/commits/0f2c4b3f24b90b9c406697b7daae7203c611f8a8